operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.1.0/ibm-spectrum-scale-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -464,15 +465,13 @@ spec:
   labels:
     operator: ibm-spectrum-scale-csi-operator
   links:
-    - name: IBM CSI Spectrum Scale Operator Documentation
-      url: https://ibm-spectrum-scale-csi-operator.readthedocs.io/en/latest/
+    - name: IBM CSI Spectrum Scale Knowledge Center
+      url: https://www.ibm.com/support/knowledgecenter/STXKQY_5.0.4/com.ibm.spectrum.scale.csi.v5r04.doc/bl1csi_kc_landing.html
     - name: CSI Developer Documentation
       url: https://kubernetes-csi.github.io/docs/
   maintainers:
     - email: jdunham@us.ibm.com
       name: John Dunham
-    - email: yadayada@in.ibm.com
-      name: Yadavendra Yadav
   maturity: alpha
```

Comment:
Is alpha the right value here?

Reply:
Only one I know will work with operator  hub right now